lib/wasix/src/state/linker.rs

```diff
@@ -519,6 +519,25 @@ pub enum LinkError {
     MissingTlsInitializer,
 }
 
+#[derive(Debug)]
+pub enum FsErrors {
+    SingleError(FsError),
```

Comment:
The naming here can be improved:

* `FsErrors` makes no sense unless you know it comes from the linker. Let's rename this to `ModuleLocationError` (location here meaning "to locate") or something similar that mentions the act of locating modules.
* `Error` in the case names is redundant; let's rename them to `Single` and `Multiple`.

Reply:
I think `LocateModuleError` makes the locate part more clear